crates/admin/src/cluster_controller/observed_cluster_state.rs

```diff
@@ -52,6 +52,11 @@ impl ObservedClusterState {
                     self.alive_nodes
                         .insert(*node_id, alive_node.generational_node_id);
                 }
+                NodeState::Suspect(maybe_node) => {
+                    self.dead_nodes.remove(node_id);
+                    self.alive_nodes
+                        .insert(*node_id, maybe_node.generational_node_id);
+                }
```

Comment:
Maybe add to the Rust doc of the `ObservedClusterState` that we treat `Suspect` as alive.